plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -80,13 +80,13 @@ public static DistributedQueryRunner create()
         return builder().build();
     }
 
-    public static Builder builder()
+    public static Builder<Builder<?>> builder()
     {
-        return new Builder();
+        return new Builder<>();
     }
 
-    public static class Builder
-            extends DistributedQueryRunner.Builder<Builder>
+    public static class Builder<B extends Builder<?>>
```

Comment:
I'd prefer a more descriptive generic param name, like `Self`.

Reply:
renamed to `SELF` - by convention generics params are SELF.